view_options/app/build.gradle.kts

```diff
@@ -69,7 +69,7 @@ dependencies {
     implementation("com.github.bumptech.glide:glide:4.13.2")
 
     // Firework SDK
-    val fireworkSdkVersion = "6.2.1"
+    val fireworkSdkVersion = "6.3.3"
```

Comment:
Hello Sir, we are already in `6.3.4`

Reply:
Good point, I will update that.
However, I am not sure if we are going to update this version per every patch release. 🤔